lib/matplotlib/axes/_axes.py

```diff
@@ -551,7 +551,7 @@ def indicate_inset_zoom(self, inset_ax, **kwargs):
         return self.indicate_inset(rect, inset_ax, **kwargs)
 
     @_docstring.dedent_interpd
-    def secondary_xaxis(self, location, *, functions=None, **kwargs):
+    def secondary_xaxis(self, location, *, functions=None, transform=None, **kwargs):
```

Comment:
This should have an entry in the Parameters section (in `_secax_docstring`).

Reply:
It's already there?